language/stdlib/modules/RegisteredCurrencies.move

```diff
@@ -52,33 +58,62 @@ module RegisteredCurrencies {
 
     /// # Module specifications
 
+    /// **Note:** I am worried in this module about the possibility that `initialize` could
+    /// be called a second time (in another module), which would reset the
+    /// currency_codes vector. I've had trouble formulating a property that
+    /// catches that possibility in this module.
```

Comment:
It seems that calling `initialize` a second time would lead to abortion, in `LibraConfig::publish_new_config_with_capability` since there is already a config under `config_account`. So `aborts_if is_initialized()` should work even without the newly added assertion in `initialize`?

Reply:
Yes, maybe you're right.  At one time, I thought T was being stored instead of LibraConfig::T<T>, and T is just a struct, not a resource, so I thought it could be overwritten.
Also, I think I tried aborts_if is_initialized() and couldn't verify it.  I'll go back and look at it more carefully.

Reply:
I checked this more carefully, and you're right.  I've removed code and specs based on this concern.